.gitignore

```diff
@@ -0,0 +1,10 @@
+.*cmd
+.*.swp
+*.o
+*.ko
+*.mod*
+Module.symvers
+modules.order
+corefreq-cli
+corefreqd
+*.keep
```

Comment:
Some of these extensions appears Distribution or Build Setup specific.
I don't have such `.*cmd ; *.swp ; *.keep` within Arch Linux; and it needs more impacts work before merge.